lib/main.dart

```diff
@@ -24,13 +24,28 @@ import "package:flutter/material.dart";
 import 'package:flutter_displaymode/flutter_displaymode.dart';
 import 'package:logging/logging.dart';
 import 'package:path_provider/path_provider.dart';
+import 'package:privacy_screen/privacy_screen.dart';
 
 final _logger = Logger("main");
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   final savedThemeMode = await AdaptiveTheme.getThemeMode();
   await _runInForeground(savedThemeMode);
+  await PrivacyScreen.instance.enable(
+    iosOptions: const PrivacyIosOptions(
+      enablePrivacy: true,
+      privacyImageName: "LaunchImage",
```

Comment:
Does this asset exist?

Reply:
it's inside the ios folder

Reply:
LaunchImage asset was already created in the ios folder

Reply:
if its not working, then we have to create it